assets/pages/new-contributions-landing/components/PaymentMethodSelector.jsx

```diff
@@ -35,9 +32,9 @@ type PropTypes = {
   currency: IsoCurrency,
   paymentMethod: PaymentMethod,
   onPaymentAuthorisation: PaymentAuthorisation => void,
-  paymentHandlers: { [PaymentMethod]: PaymentHandler | null },
+  thirdPartyPaymentLibraries: PaymentMatrix<PaymentHandler | null>,
   updatePaymentMethod: PaymentMethod => Action,
-  setPaymentIsReady: (boolean, ?{ [PaymentMethod]: PaymentHandler }) => Action,
+  setPaymentIsReady: (boolean, ?{ [Contrib]: { [PaymentMethod]: PaymentHandler }}) => Action,
```

Comment:
I think we should probably rename this function and remove the boolean since it's a bit misleading (which payment is ready?). Maybe `setThirdPartyPaymentLibrary`

Reply:
Have renamed & removed boolean, this function is only called once with 'true' and the boolean didn't affect the logic.